plugins/auth-backend/src/service/OidcRouter.ts

```diff
@@ -68,6 +98,318 @@ export class OidcRouter {
       res.json(userInfo);
     });
 
+    if (
+      this.config.getOptionalBoolean(
+        'auth.experimentalDynamicClientRegistration.enabled',
+      )
+    ) {
+      // Authorization endpoint
+      // https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest
+      // Handles the initial authorization request from the client, validates parameters,
+      // and redirects to the Authorization Session page for user approval
+      router.get('/v1/authorize', async (req, res) => {
+        // todo(blam): maybe add zod types for validating input
```

Comment:
👍 It's an auth facility after all, strictness sounds good. Same comment for all places below ofc

Do the openapi facilities don't apply for the auth backend? I guess they could. But a bit of a bigger shift to rewrite the plugin to use that.